File: python/BOJ/!_9465.py
import sys


sys.stdin = open('input.txt','r')


def solution(n,score_board):
    print(score_board)


T = int(sys.stdin.readline())

for i in range(T):
    n = int(sys.stdin.readline())
    l1 = list(map(int,sys.stdin.readline().strip().split()))
    l2 = list(map(int,sys.stdin.readline().strip().split()))
    solution(n,[l1]+[l2])

File: C/BOJ/!_9465.cpp
#include <iostream>

using namespace std;


int T,n;
int memory_arr[3][100000] = {-1};
int map_arr[3][100000] = {-1};



void printMap(int map_arr[][100000]){
    for (int i=0;i<3;i++){
        for (int j=0;j<n;j++){
            cout << map_arr[i][j] << "\t";
        }
        cout << endl;
    }
}


int solution(){
    printMap(map_arr);
}


int main(){
    ios::sync_with_stdio(false);
    cin.tie(0);
    
    freopen("input.txt", "r", stdin);  
    
    cin >> T;
    for (int i=0;i<T;i++){
        cin >> n;
        for(int j=0;j<2;j++){
            for(int k=0;k<n;k++){
                cin >> map_arr[j][k];
            }
        }
        printMap(map_arr);
    }



    return 0;
}







